build.sbt

```diff
@@ -444,6 +445,32 @@ lazy val `Tools` = (project in file("DEVELOPING_OPAL/tools"))
   .dependsOn(framework % "it->it;it->test;test->test;compile->compile")
   .configs(IntegrationTest)
 
+lazy val ce = (project in file("TOOLS/ce"))
+  .dependsOn(br % "compile->compile")
+  .settings(buildSettings: _*)
+  .settings(
+    fork := true,
+    javaOptions += s"-Dbuild.version=${version.value}",
+    name := "ce",
+    libraryDependencies ++= Dependencies.ce,
+    Compile / doc := {
+      // Overrides doc method to include config documentation at doc
+      val originalDoc = (Compile / doc).value
+      (Compile / compile).value
+      (Compile / run).toTask("").value
+      originalDoc
+    },
+    Compile / doc / scalacOptions ++= Opts.doc.title("OPAL - Configuration Explorer")
+  )
+.dependsOn(
+  br % "compile->compile",
```

Comment:
Add your new project to the ProjectDependencies.mmd file and regenerate the pdf/svg from it

Reply:
Done

Reply:
SVG still missing

Reply:
Fixed it now